packages/map-template/src/helpers/GetLegendSectionsHeight.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Get height of the legend sections.
+ * Returns a promise with the height in pixels.
+ *
+ * @returns {Promise}
+ */
+export default function getLegendSectionsHeight() {
```

Comment:
This is related to the LegendDialog component only. I think this file should be moved into the LegendDialog folder.

Reply:
✅